src/Microsoft.VisualStudio.Web.CodeGeneration.Msbuild/Microsoft.VisualStudio.Web.CodeGeneration.Msbuild.csproj

```diff
@@ -4,7 +4,7 @@
 
   <PropertyGroup>
     <Description>MSBuild task (EvaluateProjectInfoForCodeGeneration) used by Microsoft.VisualStudio.Web.CodeGeneration.Tools</Description>
-    <TargetFrameworks>net461;netstandard1.6</TargetFrameworks>
+    <TargetFrameworks>net461;netstandard2.0</TargetFrameworks>
```

Comment:
It should. We have other tasks that do. In the 2.0.0 SDK, MSBuild is a .NET Core 2.0 app.